core/plugin/zplugin.go

```diff
@@ -15,6 +15,7 @@ import (
 	_ "github.com/coredns/coredns/plugin/erratic"
 	_ "github.com/coredns/coredns/plugin/errors"
 	_ "github.com/coredns/coredns/plugin/etcd"
+	_ "github.com/coredns/coredns/plugin/fallback"
```

Comment:
no reason to put it here? Can bump it to its own repo.

Reply:
yeah, we could do that. @yuewko I will create a separate repo for it in CoreDNS org.